guides/common/modules/proc_disabling-puppet.adoc

```diff
@@ -0,0 +1,26 @@
+[id="Disabling_Puppet_Integration_{context}"]
+= Disabling Puppet Integration with {Project}
+
+To discontinue using Puppet in your {Project}, follow this procedure.
+
+.Prerequisites
+* You upgraded {Project} from the previous version with Puppet server enabled.
+
+.Procedure
+. Disable Puppet server on {SmartProxies} before you disable it on {ProjectServer}:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-maintain} plugin purge-puppet
+----
++
+If your deployment includes a database on {SmartProxy}, the command will also remove internal data from the Puppet database.
```

Comment:
I don't think database on SmartProxy is really a thing. While there is PuppetDB, it's an entirely optional project and we don't support it. The removal procedure thus doesn't remove PuppetDB.

It really removes `/etc/puppetlabs` and `/opt/puppetlabs/server/data`. So in the phrasing I'd keep it to `data`, not `database`.

However, looking at the code now it also calls `foreman-rake purge:puppet` which certainly does NOT work on a SmartProxy so right now it can't work on a SmarProxy.

We'd need to look into that but for now I think this line should be dropped:
```suggestion
```

Reply:
I agree, the command will not remove the database, because there is no "internal" data (as in internal to Foreman/Satellite) held here.

Reply:
Thank you both! Dropping line 17.